app/ide-desktop/lib/types/modules.d.ts

```diff
@@ -2,6 +2,23 @@
  *
  * This file MUST NOT `export {}` for the modules to be visible to other files. */
 
+declare module '*/build.json' {
+    interface BuildInfo {
+        commit: string
+        version: string
+        engineVersion: string
+        name: string
+    }
+
+    const BUILD_INFO: BuildInfo
+    export default BUILD_INFO
+}
+
+declare module '*/ensogl-pack/linked-dist' {
+    // eslint-disable-next-line no-restricted-syntax
+    export * from '../../../../lib/rust/ensogl/pack/js/src/runner/index'
+}
```

Comment:
What does it do and why it is better than the previous version? I'm asking just to understand what is the difference here.

Reply:
1. i have no clue whether this is a proper fix - but the idea is that this removes the dependency on the built module, so we are now pointing at the actual source (which always exists) instead of the built `.d.ts`
2. i don't think so - as mentioned elsewhere i think it only happens on clean runners (?!). the issue happens on the typecheck run, so what's happening is, typescript is looking for the type definitions but cannot find them